_posts/2019-05-16-let-the-dao-work.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Let the DAO work"
```

Comment:
Maybe make title something like "Time for the DAO to take over"?

Reply:
As it is now the title seems to imply the DAO isn't working, in a functional way.